<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecordForm</class>
 <widget class="QDialog" name="AddRecordForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>234</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>234</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>234</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>241</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>281</width>
     <height>154</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>First Name:</string>
        </property>
        <property name="buddy">
         <cstring>FirstName_LE</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Last Name:</string>
        </property>
        <property name="buddy">
         <cstring>LastName_LE</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Salary:</string>
        </property>
        <property name="buddy">
         <cstring>Salary_LE</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Hiring Year:</string>
        </property>
        <property name="buddy">
         <cstring>HiringYear_LE</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="FirstName_LE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="LastName_LE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Salary_LE"/>
      </item>
      <item>
       <widget class="QLineEdit" name="HiringYear_LE"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FirstName_LE</tabstop>
  <tabstop>LastName_LE</tabstop>
  <tabstop>Salary_LE</tabstop>
  <tabstop>HiringYear_LE</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddRecordForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
